backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/scheduling/SchedulingManager.java

```diff
@@ -431,7 +432,8 @@ private Map<Guid, Guid> schedule(Cluster cluster,
                         .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue, NumaNodeMemoryConsumption::merge));
                 updateHostNumaNodes(host, numaConsumption);
 
-
+                // Sorting the VMs to get better dedicated CPU pinning.
```

Comment:
what's the reason for sorting the vms both here and in CpuPinningPolicyUnit?

Reply:
dropped this part, the sorting is now separate in PR #384